docs/building.md

```diff
@@ -64,6 +64,10 @@ type custom_ep_param =
   | MyCustomEpDummy of unit
 ```
 
+Next we should define the contract-extra field for the variant as well. This can be done by copying
+and the `src/template_storage.mligo` to `src/acmeDAO_storage.mligo` and modifying it to contain the
+actual type and a default value for the contract-extra field for this variant.
+
```

Comment:
Aside from some of the comments I left above, I think that even more updates are needed to many of the docs in the `docs` dir.

AFAICT a lot of details that were explained there are no longer correct.
Would you please check them carefully and update them accordingly?

Reply:
Have have checked all the docs in `docs` dir and updated where it seemed required.